02-oop-inheritance/javaScript/refactored_JS/main  v0.1.js

```diff
@@ -0,0 +1,44 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function ( title ) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function( title ) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+var Movie = function (paramName, paramValue) {
```

Comment:
also this constructor definition is not right, you should provide as example name or title, genre, time, etc. Not a parameter name and a value. As I asked before what happens if I want to initialize 3 parameters on the constructor?